pip_audit/_cli.py

```diff
@@ -380,11 +387,16 @@ def audit() -> None:  # pragma: no cover
             parser.error("The --extra-index-url flag can only be used with --requirement (-r)")
         elif args.no_deps:
             parser.error("The --no-deps flag can only be used with --requirement (-r)")
+        elif args.disable_pip:
+            parser.error("The --disable-pip flag can only be used with --requirement (-r)")
 
     # Nudge users to consider alternate workflows.
     if args.require_hashes and args.no_deps:
         logger.warning("The --no-deps flag is redundant when used with --require-hashes")
 
+    if args.no_deps and args.disable_pip:
+        logger.warning("The --no-deps flag is redundant without --disable-pip")
```

Comment:
Minor typo? `without -> with`:

```suggestion
        logger.warning("The --no-deps flag is redundant when used with --disable-pip")
```